front/app/components/ParticipationCTABars/IdeationCTABar/index.tsx

```diff
@@ -78,11 +79,23 @@ const IdeationCTABar = ({ phases, project }: CTABarProps) => {
             fontWeight="500"
             bgColor={theme.colors.white}
             textColor={theme.colors.tenantText}
-            textHoverColor={theme.colors.black}
+            bgHoverColor={theme.colors.grey100}
             padding="6px 12px"
             fontSize="14px"
           >
-            <FormattedMessage {...messages.seeIdeas} />
+            <FormattedMessage
+              {...getInputTermMessage(getInputTerm(phases), {
+                idea: messages.seeIdeas,
+                option: messages.seeOptions,
+                project: messages.seeProjects,
+                question: messages.seeQuestions,
+                issue: messages.seeIssues,
+                contribution: messages.seeContributions,
+                // proposal: messages.seeProposals,
```

Comment:
I will uncomment this as soon as we merge the input term for proposals PR which is still waiting for translations.

Reply:
@IvaKop Do we need to have it commented out if we don't have any projects yet with proposal/initiative/petitions? :thinking:

Reply:
It's commented out because this list is typed and we need to merge the new options for the input term first so that we don't get a type error.

Reply:
Ahh gotcha - ok clear! :)